scripts/propose.py

```diff
@@ -0,0 +1,6 @@
+import grpc
+from rchain.client import RClient
+
+with grpc.insecure_channel('localhost:40401') as channel:
+	client = RClient(channel)
+	client.propose
```

Comment:
I guess in this file we can implement https://rchain.atlassian.net/browse/RCHAIN-3832 by spinning out a subprocess waiting for proposed block to be finalized, and writing to influxdb when it is @zsluedem WDY think, does this make sense?

Reply:
You can do that, too. Let me assign all those tickets to you.